### DISCUSSION TITLE
Unknown error later of UWP

### DISCUSSION THREAD

**Discussion by CarlosPortillo11** [farag2/Sophia-Script-for-Windows#279] [Q&A]
Hello, I was making all the process right the way, later of the UWP Prompt and uninstall the unnecessary things, this error show from nowhere, don't know how to fix and really want to.  I'll be really grateful to any help.

![image_2021-10-24_190625](https://user-images.githubusercontent.com/47508370/138620516-e4d7435f-97b0-493b-a823-437d815c416a.png)

**Comment by farag2** [comment]
Hi. Your Windows version?
